MapboxNavigation.podspec

```diff
@@ -51,4 +51,13 @@ Pod::Spec.new do |s|
 
   s.swift_version = "5.0"
 
+  # https://github.com/mapbox/mapbox-navigation-ios/issues/2665
+  s.user_target_xcconfig = {
+    'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => '$(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT)__XCODE_$(XCODE_VERSION_MAJOR))',
+    'EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64__XCODE_1200' => 'arm64 arm64e armv7 armv7s armv6 armv8'
+  }
```

Comment:
If I remove this bit, then I’m able to push to CocoaPods trunk: CocoaPods/Specs@f966ab4195f6b867cff6fa6281635bb671d0ce4c.

```
$ pod trunk push test-1ec5-MapboxNavigation.podspec --allow-warnings
Updating spec repo `trunk`
Validating podspec
 -> test-1ec5-MapboxNavigation (1.0.0+2)
    - WARN  | source: The version should be included in the Git tag.
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Building targets in parallel
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | [iOS] xcodebuild:  note: Planning build
    - NOTE  | [iOS] xcodebuild:  note: Constructing build description
    - NOTE  | [iOS] xcodebuild:  warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'Polyline' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'MapboxMobileEvents' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
    - NOTE  | xcodebuild:  note: detected encoding of input file as Unicode (UTF-8) (in target 'test-1ec5-MapboxNavigation' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'Solar' from project 'Pods')

Updating spec repo `trunk`

--------------------------------------------------------------------------------
 🎉  Congrats

 🚀  test-1ec5-MapboxNavigation (1.0.0+2) successfully published
 📅  September 30th, 21:54
 🌎  https://cocoapods.org/pods/test-1ec5-MapboxNavigation
 👍  Tell your friends!
--------------------------------------------------------------------------------
```